compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -250,17 +250,34 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                 .map_or_else(String::new, |res| format!("{} ", res.descr()));
                 (mod_prefix, format!("`{}`", Segment::names_to_string(mod_path)), None)
             };
+
+            let (fallback_label, override_sugg) = if path_str == "async"
```

Comment:
```suggestion
            let (fallback_label, suggestion) = if path_str == "async"
```
nit: I think it's fine to just shadow the name here.

Reply:
Smart think, yes!